ImportError: cannot import name '_check_google_client_version' from 'pandas_gbq.gbq'

**What happened**:

`pandas-gbq` released version [0.15.0](https://github.com/pydata/pandas-gbq/releases/tag/0.15.0) which broke `apache-airflow-backport-providers-google==2020.11.23`

```
../lib/python3.7/site-packages/airflow/providers/google/cloud/hooks/bigquery.py:49: in <module>
    from pandas_gbq.gbq import (
E   ImportError: cannot import name '_check_google_client_version' from 'pandas_gbq.gbq' (/usr/local/lib/python3.7/site-packages/pandas_gbq/gbq.py)
```

The fix is to pin `pandas-gpq==0.14.1`.

--- a/setup.py
+++ b/setup.py
@@ -313,7 +313,7 @@ google = [
     'google-cloud-workflows>=0.1.0,<2.0.0',
     'grpcio-gcp>=0.2.2',
     'json-merge-patch~=0.2',
-    'pandas-gbq',
+    'pandas-gbq<0.15.0',
     'plyvel',
 ]
 grpc = [

